packages/sdk-auth/src/tokenProvider.js

```diff
@@ -91,6 +93,8 @@ export default class TokenProvider {
     return this.refreshTokenFlowPromise.then((refreshTokenInfo) => {
       this.refreshTokenFlowPromise = null
       return refreshTokenInfo
+    }).catch(() => {
+        this.refreshTokenFlowPromise = null
```

Comment:
```suggestion
    }).catch((e) => {
        this.fetchTokenInfoPromise = null
        throw e
```
Just swallowing the error looks dangerous to me, how about above?

Reply:
C&P error as your suggestion resets the wrong promise

Reply:
Didn't realize and even that resetting the promise so updated the code.

Reply:
Interesting, what does C&P error mean, @jenschude?

Reply:
Copy&Paste. I meant your suggestion proposes instead of refreshTokenPromise a fetchTokenPromise

Reply:
Ooops. Good one 👍